events/bingo/bingoBossApp/ui/css/bingoBossApp_ui.css

```diff
@@ -0,0 +1,109 @@
+*, *:before, *:after {
+    -webkit-box-sizing: inherit;
+    -moz-box-sizing: inherit;
+    box-sizing: inherit;
+}
+
+html {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Raleway', sans-serif;
+    color: #cccbcb;
+    background-color: #2b2b2b;
+    overflow: hidden;
+    margin: 0;
+    padding: 0;
+}
+
+#titleBarContainer {
+    display: flex;
```

Comment:
Every time I learn flex, I forget it.  This is one I need to work on more for sue.  Thanks for using it here.

Reply:
this guide is extremely useful; i regularly consult it:
https://css-tricks.com/snippets/css/a-guide-to-flexbox/